common/blockchain/src/manager.rs

```diff
@@ -289,6 +289,8 @@ impl<PK: crate::PubKey> ChainManager<PK> {
     }
 
     pub fn epoch_height(&self) -> crate::HostHeight { self.epoch_height }
+
+    pub fn genesis(&self) -> CryptoHash { self.genesis.clone() }
```

Comment:
```suggestion
    pub fn genesis(&self) -> &CryptoHash { &self.genesis }
```

Caller can decide whether they want to clone.

Reply:
gotcha